app/package/models.py

```diff
@@ -122,16 +122,14 @@ def build_s3_object_url(self, domain_name, path):
                    key=self.build_s3_key(path))
 
     def generate_pre_signed_post_object(self, path, md5,
```

Comment:
Is there no change to a controller to reflect this change? Surely the API docs need to change?

Also what about tests?